### PR TITLE
Added Sum of Multiples exercise

### PR DESCRIPTION
**Pull request by ryanplusplus**
[Exercise description](https://github.com/exercism/x-common/blob/master/sum-of-multiples.md)

### CODE REVIEW

File: exercises/sum-of-multiples/example.lua [hunk 1]
```diff
@@ -0,0 +1,24 @@
+return function(numbers)
+  numbers = numbers or { 3, 5 }
+
+  return {
+    to = function(limit)
```
Comment (aarti) [inline exercises/sum-of-multiples/example.lua]: might be hard for users new to lua to grasp this chaining pattern of named returns

**Comment by aarti** [comment]
Other than that :+1:  looks great

File: exercises/sum-of-multiples/example.lua (hunk 1 above)
Comment (ryanplusplus) [inline exercises/sum-of-multiples/example.lua]: Okay, so basically just put it further down in `config.json`?

Comment (aarti) [inline exercises/sum-of-multiples/example.lua]: If someone has a hard time we can move it further down, it is already at 20 or so.

**Pull request merged**